lib/mix/tasks/phoenix.new.ex

```diff
@@ -33,6 +34,12 @@ defmodule Mix.Tasks.Phoenix.New do
 
     copy_from template_dir, path, application_name, &EEx.eval_file(&1, binding)
     copy_from static_dir, Path.join(path, "priv/static"), application_name, &File.read!(&1)
+
+    # TODO decide to auto npm install or not
+    if npm_path && Mix.env == :dev do
```

Comment:
I am OK with installing it. I think though that, 

1. Should we provide an option to disable npm install?

2. Should we provide an option to disable brunch?

3. If NPM is not available, I think we should emit a message to the user: "You don't have npm installed. This means we were not able to install all packages... yada yada... install npm and run npm instal inside the app"